src/wallet/wallet_rpc_server.cpp

```diff
@@ -4708,6 +4708,7 @@ int main(int argc, char** argv) {
   command_line::add_arg(desc_params, arg_prompt_for_password);
   command_line::add_arg(desc_params, arg_rpc_client_secret_key);
   command_line::add_arg(desc_params, arg_no_initial_sync);
+  command_line::add_arg(desc_params, daemonizer::arg_non_interactive);
```

Comment:
> daemonizer still needs to be able to read the param from the vm or it will crash

You sure about this? I can't see where the dependency is. In any case, adding in the hidden_options like you have a moment ago seems reasonable.

Reply:
Thanks, you _should_ be able to drop this line. That said, I suppose it does no harm leaving like this either.